akka-actor-typed/src/main/scala/akka/actor/typed/internal/pubsub/TopicImpl.scala

```diff
@@ -33,7 +33,7 @@ private[akka] object TopicImpl {
 
   // internal messages, note that the protobuf serializer for those sent remotely is defined in akka-cluster-typed
   final case class GetTopicStats[T](replyTo: ActorRef[TopicStats]) extends Topic.Command[T]
-  final case class TopicStats(localSubscriberCount: Int, topicInstanceCount: Int)
+  final case class TopicStats(localSubscriberCount: Int, topicInstanceCount: Int) extends Topic.TopicStats
```

Comment:
why don't we move `GetTopicStats` and `TopicStats` classes to the public `Topic`?

Reply:
There's a note a few lines up about keeping actual protocol secret for easier bincomp evolve. But maybe less likely that we'd want to add some fields for these messages?

Reply:
let's keep the current design